test/norm_cast.lean

```diff
@@ -0,0 +1,81 @@
+import Std.Tactic.NormCast
+import Std
+
+/-!
+# Tests for norm_cast
+
+Since the behaviour of `norm_cast` depends on the `@[norm_cast]` attribute being applied correctly,
+this test file imports all of `Std` as an integration test.
```

Comment:
I'd prefer you didn't. Just import the specific files containing `@[norm_cast]` attributes executed in the test.

Reply:
done.